mobile_app/collaboration/navigation/navigator.js

```diff
@@ -24,21 +22,15 @@ function Navigator() {
         <Stack.Screen name="CameraPage" component={CameraPage} options={{ headerShown: false }} />
         <Stack.Screen name="HomePage" component={HomePage} options={{ headerShown: false }} />
         <Stack.Screen name="CameraFeature" component={CameraFeature} options={{ headerShown: false }}/>
+        <Stack.Screen name="Profile" component={Profile} options={{ headerShown: false}}/>
         <Stack.Screen name="Result" component={Result} options={{ headerShown: false}}/>
-        <Stack.Screen name="HealthResult" component={HealthResult} options={{ headerShown: false}}/>
```

Comment:
Will the page work if this line is deleted

Reply:
yes